lib/matplotlib/projections/polar.py

```diff
@@ -521,9 +523,17 @@ def set_thetagrids(self, angles, labels=None, frac=None, fmt=None,
         ACCEPTS: sequence of floats
         """
         # Make sure we take into account unitized data
-        angles = self.convert_yunits(angles)
-        angles = np.asarray(angles, np.float_)
-        self.set_xticks(angles * (np.pi / 180.0))
+        if isinstance( angles[0], float ):
```

Comment:
Why do we need this if-branch? Is `convert_yunits()` not returning radians? If not, is that a regression?